data/arrrrcamp/arrrrcamp-2015/videos.yml

```diff
@@ -0,0 +1,233 @@
+    of ideas and tools that can help you make your application more stable and more
+    resilient to the failure of external services, an issue that is often neglected
+    until it's too late. More importantly, we will talk about how you can write meaningful
+    and realistic integration tests and set up \"chaos testing environments\" to gain
+    confidence in your application's resiliency."
+  video_id: mTIO9fj1UIc
+
+- title: Developing Ruby Applications with Docker
+  raw_title: ArrrrCamp 2015 - Developing Ruby Applications with Docker by  Laura Frank
+  speakers:
+    - Laura Frank
+  event_name: ArrrrCamp 2015
+  published_at: '2015-10-01'
+  description: |-
+    Docker’s lightweight virtualization may supplant our hypervisor-backed VMs at some point in the future, and change the way that tomorrow's Ruby applications are architected, packaged and deployed. Using Docker, your applications will sit atop an excellent platform for packing, shipping and running low-overhead, isolated execution environments. You will get a brief intro to the Docker ecosystem, get to know the tools and processes needed to create containerized applications, and learn best practices for interacting with the Docker API and CLI.
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/H4Of/
+  video_id: hiiPMgZbzcA
+
+
+# LUNCH
+
+- title: 'Peeking into Ruby: Tracing Running Code'
+  raw_title: 'ArrrrCamp 2015 - Peeking into Ruby: Tracing Running Code by Jason Clark'
+  speakers:
+    - Jason Clark
+  event_name: ArrrrCamp 2015
+  published_at: '2015-10-01'
+  description: |-
+    Your Ruby app is in production, but something isn't quite right. It worked locally, it passed CI... yet the running app is acting up. Sounds familiar? You're in luck! Multiple tools exist for grappling with a running Ruby app. This talk will introduce a variety of tools and techniques for peeking into what your Ruby app is doing.
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/H4Oe/
+  video_id: SBtJ7da_rA8
+
+- title: 'Ruby keyword args and the options hash, from the parser to the virtual machine'
+  raw_title: ArrrrCamp 2015 - Ruby keyword args and the options hash, from the parser to the virtual machine
+  speakers:
+    - Étienne Barrié
+  event_name: ArrrrCamp 2015
+  published_at: '2015-10-01'
+  description: |-
+    By Étienne Barrié
+    Ruby has slowly but surely added support for keyword arguments. Starting from the implicit braces for a hash at the end of an argument list, it has grown up to required keyword arguments in 2.1. This talk will try to convince you that keyword arguments are a lie and don't even exist, and why you should use them anyway.
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/H4Od/
+  video_id: e4t9dts_07g
+
+- title: Lightning Talks (Day 1)
+  raw_title: ArrrrCamp 2015 - Day 1 LIGHTNING TALKS
+  speakers:
+    # - Yves # TODO: missing last name (https://cln.sh/rs3gNcwS) # Walking (00:20)
```

Comment:
@joren maybe you can help us identify this person's last name, thank you! 🙌🏼

Reply:
Yes, that is Yves Hanoulle

Reply:
Awesome, thank you! Fixed in https://github.com/adrienpoly/rubyvideo/commit/99fbdaf448fc621fbb1a0604ed7dd6cf2067a749